docs/candyMachine/index.md

```
@@ -0,0 +1,603 @@
+- **Allow List**: Restrict to specific wallets
+- **Freeze Sol Payment**: Freeze SOL for payment
+- **Freeze Token Payment**: Freeze tokens for payment
+
+### Guard Groups
+
+Guard groups allow you to create different minting phases with distinct rules and restrictions. Each group can have its own set of guards, enabling complex minting strategies like whitelist phases, public sales, and exclusive drops.
+
+For comprehensive information about guard groups, see [Guard Groups Documentation](./groups.md).
+
+**Example Group Configuration:**
+```json
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)